app/views/course/virtual_classrooms/_virtual_classroom.html.slim

```diff
@@ -27,9 +27,10 @@
   div.access-link
     - link_exists = virtual_classroom.instructor_classroom_link.present?
     - is_instructor = can?(:edit, virtual_classroom)
+    - is_main_instructor = virtual_classroom.instructor == current_user
```

Comment:
It's not necessarily true. There are 3 possible scenarios:
1. An `instructor_classroom_link` has been generated by current user
2. An `instructor_classroom_link` has been generated by a different admin
3. No `instructor_classroom_link` has been generated

In case 2, `is_main_instructor` is `false` but `link_exists` is `true`